### ISSUE TITLE
Intermittent garbled audio / no video ("loud robot bug")

### ISSUE THREAD

**Issue by dbkr** [vector-im/element-call#1893]
### Steps to reproduce

We are seeing a bunch of reports where a user will connect (sometimes after disconnecting due to bad network) and then appear to all or some users in the call as "loud robot" noises (sounds like the decoder trying to decode gibberish, ie. trying to decode an encrypted audio stream).

This seemed to start after the 0.5.10 release which included per-user key e2e, however this should not be enabled for any calls on the SPA. Also, all the logs I've seen see manageMediaKeys=false (ie. per user key mode is off) and the room I looked in had no media key events. This could be a red herring.

I have also not seen a single decryption error in all the logs I've looked at, which suggests livekit believes it is decrypting media successfully with a valid key. We tested by deliberately setting a bad password and decryption errors do come through to a rageshake log.

We did also update the livekit client SDK in v0.5.10. We have not yet updated the server on live (we're now 2 versions behind).

### Will you send logs?

Yes

**Comment by dbkr** [comment]
We reverted the PR for per-user keys (https://github.com/vector-im/element-call/pull/1772) in the 0.5.11 release. The bug re-occurred on a call after this on Wednesday evening. We have yet to see the bug occur with the v0.5.9 release which we rolled back to after the bug happened again. Unfortunately we only have one rageshake from this reoccurrence.

The Weds reoccurrence of the bug was in a room created by our calendar addon, so we believed that this could have something to do with it. The calendar addon creates its own room and also generates the e2e password itself, however in the rageshakes, both parties do have e2ee enabled so that rules out a bug where one had managed to decide the room was e2e and one not. In any case, this would require one client to think the room had a canonical alias, which it doesn't.

The key is just set as a string to livekit, so any difference in formatting here seems unlikely. We also don't observe any decryption errors in the rageshake, which suggests that the clients seeing the loud robots are not trying to decrypt with an incorrect key. It could be that they are not in e2e mode at all and the loud robots are.

We tried to repro the bug on the dev env after the reoccurrence on Weds and could not repro it. We are therefore working on the assumption that the bug does not occur with the current livekit branch HEAD and have done another release (0.5.12) to test this theory.
